libkbfs/kbfs_ops.go

```diff
@@ -754,6 +755,35 @@ func (fs *KBFSOpsStandard) GetNodeMetadata(ctx context.Context, node Node) (
 	return ops.GetNodeMetadata(ctx, node)
 }
 
+// TeamNameChanged implements the KBFSOps interface for KBFSOpsStandard
+func (fs *KBFSOpsStandard) TeamNameChanged(
+	ctx context.Context, tid keybase1.TeamID) {
+	fs.log.CDebugf(ctx, "Got TeamNameChanged for %s", tid)
+	fs.opsLock.Lock()
+	defer fs.opsLock.Unlock()
+	// We have to search for the tid since we don't know the old name
+	// of the team here.  TODO: add an index for this?
```

Comment:
This is a question, not a real TODO yet.  I'm not comfortable littering JIRA with something I'm not sure is a good idea.  I'll remove the "TODO" label and just make it a question if you prefer.